Modules/SubModule_Import/xASL_bids_AddGeneratedByField.m

```diff
@@ -0,0 +1,42 @@
+function xASL_bids_AddGeneratedByField(x, pathJSONin, pathJSONout)
+%xASL_bids_AddGeneratedByField Add the generated by field to the struct.
+%
+% FORMAT:     xASL_bids_AddGeneratedByField(x, pathJSONin, pathJSONout)
+% 
+% INPUT:      x           - ExploreASL x struct (STRUCT, REQUIRED)
+%             pathJSONin  - Path to input JSON file (STRING, REQUIRED)
+%             pathJSONout - Path to output JSON file (STRING, REQUIRED)
```

Comment:
This last variable is not required. If nargin =2, then overwrite the input - see line 22 and modify header accordingly.

Reply:
Easy fix 77a1e39 👍